src/containers/Editor.js

```diff
@@ -19,27 +20,41 @@ class Editor extends React.Component {
   componentDidMount(){
     getDocument(1).then(
       data => this.setState({ defaultJSON: data })
-    );
+    ).catch (() => {
+      this.setState({ defaultJSON: "", serverRunning: false })
+    })
   }
+  
   render = () => {
     if (this.state.defaultJSON == null){
       return(<div>Loading...</div>)
     }else{
       console.log("Actual Editor")
       const {body} = document;
       if (body) body.style.backgroundColor = this.props.editorDarkMode ? '#181A1B' : '#FFF';
-      return (
-        <RichMarkdownEditor
-          readOnly={this.props.editorReadOnly}
-          dark={this.props.editorDarkMode}
-          key={this.props.fileNameKey}
-          // defaultValue={localStorage.getItem(FILE_NAME_PREFIX_LOCAL_STORAGE_KEY + this.props.fileNameKey) || ''}
-          defaultJSON = {this.state.defaultJSON}
-          tagFilters={this.props.tagFiltersExpr}
-          onChange={this.handleEditorChange}
-          onSave={options => putDocument(options['doc'].toJSON(), 1)}
-        />
-      );
+      if(this.state.serverRunning){
+        return (
+          <RichMarkdownEditor
+            readOnly={this.props.editorReadOnly}
+            dark={this.props.editorDarkMode}
+            key={this.props.fileNameKey}
+            defaultJSON = {this.state.defaultJSON}
+            tagFilters={this.props.tagFiltersExpr}
+            onSave={options => putDocument(options['doc'].toJSON(), 1)}
+          />
+        );
+      }else{
+        return (
+          <RichMarkdownEditor
+            readOnly={this.props.editorReadOnly}
+            dark={this.props.editorDarkMode}
+            key={this.props.fileNameKey}
+            defaultValue={localStorage.getItem(FILE_NAME_PREFIX_LOCAL_STORAGE_KEY + this.props.fileNameKey) || ''}
```

Comment:
Is there a way to use `defaultJSON` for both so that we just have the `if` `else` modify some variable passed into `RichMarkdownEditor`?